src/shared/lib/StoreUtils.spec.ts

```diff
@@ -288,10 +288,20 @@ describe('StoreUtils', () => {
         });
     });
 
-    it("won't fetch onkokb data if there are no mutations", (done) => {
-        fetchOncoKbData({}, emptyMutationData).then((data: any) => {
-            assert.deepEqual(data, {sampleToTumorMap: {}, indicatorMap: {}});
-            done();
+    describe('fetchOncoKbData', () => {
+        it("won't fetch onkokb data if there are no mutations", (done) => {
+            fetchOncoKbData({}, emptyMutationData).then((data: any) => {
```

Comment:
if you wanted to avoid using done, you may be able to use the fake clock library `lolex`. It makes the test run a bit faster. Not super necessary though. It's already in package.json, I'm pretty sure it's used in the `accumulatingDebounce` tests